portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -9,15 +9,15 @@
  * @author Olga Shimanskaia <olgashimanskaia@gmail.com>
  */
 public class Comments {
-    private static final String COMMENT_ENTITY_KIND = "Comment";
+    public static final String COMMENT_ENTITY_KIND = "Comment";
     public static final String DATE = "date";
     public static final String RATING = "rating";
     public static final String TEXT = "comment-text";
 
     /**
      * Represents the comment.
      */
-    private static class Comment {
+    public static class Comment {
         Date date;
         String text;
         long rating;
```

Comment:
This fields anyway will be protected, so it's still not possible to access them externally, but it's a good point, thank you, I will make them private, because this class isn't expected to be extended.